requirements.txt

```diff
@@ -8,3 +8,4 @@ python-dotenv
 pydantic
 requests
 termcolor
+boto3
```

Comment:
this is not needed here?

Reply:
Wasn't sure at first, but this will be brought in registry part. I'll remove.